pykechain/models/scope2.py

```diff
@@ -112,7 +110,7 @@ def _update_scope_project_team(self, select_action, user, user_type):
                                          data={'user_id': user_object['pk']})
 
         if response.status_code != requests.codes.ok:  # pragma: no cover
-            raise APIError("Could not {} {} in Scope".format(select_action.split('_')[0], user_type), response=response)
+            raise APIError("Could not {} {} in Scope".format(*select_action.split('_')), response=response)
```

Comment:
need to be sure that the select_action is properly provided and according to the ENUM to work this.